### PR TITLE
[Merged by Bors] - Set resource limits by default

### PR DESCRIPTION
**Pull request by soenkeliebau**
# Description

Needed for: 
https://github.com/stackabletech/issues/issues/368

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes
 
```[tasklist]
# Author
- [ ] Changes are OpenShift compatible
- [ ] CRD changes approved
- [ ] Helm chart can be installed and deployed operator works
- [ ] Integration tests passed (for non trivial changes)
```

```[tasklist]
# Reviewer
- [ ] Code contains useful comments
- [ ] (Integration-)Test cases added
- [ ] Documentation added or updated
- [ ] Changelog updated
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
```

```[tasklist]
# Acceptance
- [ ] Feature Tracker has been updated
- [ ] Proper release label has been added
```

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ All notable changes to this project will be documented in this file.
 - Use 0.0.0-dev product images for testing ([#441]).
 - Use testing-tools 0.2.0 ([#441]).
 - Added kuttl test suites ([#455])
+- Set resource limits on containers by default ([#456])
```
Comment (sbernauer) [inline CHANGELOG.md:20]: I would suggest to rename the entry above to `- Set explicit resources on all containers ([#453], [#456]).`

**Review by sbernauer — APPROVED** [review]
LGTM % comment

File: CHANGELOG.md (hunk 1 above)
Comment (soenkeliebau) [reply CHANGELOG.md]: changed

**Pull request closed without merging**